.travis.yml

```diff
@@ -33,7 +33,7 @@ matrix:
             - ubuntu-toolchain-r-test
 
     # ubuntu 14.04, gcc-6
-    - env: ["VER=6"]
+    - env: ["VER=6", "EXT=YES"]
```

Comment:
Just curious, any particular reason that `EXT=YES` is attached to `gcc-6` instead of other GCC versions? Is it because it is the next version in the list that doesn't have extra envs added to it, since `gcc-5` already have `COV=YES`?

Reply:
Erm nope I am not sure if you have misunderstood my intent; I am fine with just `gcc-6` + `EXT=YES`, and would prefer it to stay that way and not want to have `EXT=YES` for all the other environments.

Please revert the changes to `.travis.yml` in commit `ed55d95`, but keep the shifting of `cpptoml_config.h.in ` with the amendment of `CMakeLists.txt`.

The other changes look good to me.